UI.py

```diff
@@ -224,12 +224,15 @@ async def UpdateEmbedLeaveStatus(message, embed, newStatus):
 
     await message.edit(embed=embed)
 
-async def UpdateEmbedApprovedRejectedby(message, embed, member):
+async def UpdateEmbedApprovedRejectedby(message, embed, member, is_reverted):
     embed_dict = embed.to_dict()
 
     for field in embed_dict["fields"]:
         if field["name"].lower() == "approved/rejected by":
-            field["value"] = f'<@!{member.id}>'
+            if is_reverted:
+                field["value"] = "None"
+            else:
+                field["value"] = f'<@!{member.id}>'
```

Comment:
I think these changes are unnecessary, the embed's new status can be reverted and we can keep the tagging of the person who reverted it.

Reply:
Done